app/src/main/java/org/pesmypetcare/mypetcare/utilities/ImageManager.java

```diff
@@ -0,0 +1,124 @@
+package org.pesmypetcare.mypetcare.utilities;
+
+import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.os.Environment;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ImageManager {
+    public static final String PET_PROFILE_IMAGES_PATH = "petProfileImages";
+    public static final String USER_PROFILE_IMAGES_PATH = "userProfileImages";
+    private static final String PATH = Environment.getExternalStorageDirectory().getAbsolutePath()
+        + File.separator + "MyPetCare";
+    private static final String EXTENSION = ".jpg";
+    private static final int QUALITY = 100;
+    private static Bitmap defaultPetImage;
+    private static byte[] defaultBytesPetImage;
+
+    private ImageManager() {
+
+    }
+
+    /**
+     * Get the bytes of the bitmap image.
+     * @param image The bitmap that the bytes has to be obtained
+     * @return The bytes of the bitmap
+     */
+    public static byte[] getImageBytes(Bitmap image) {
+        ByteArrayOutputStream stream = new ByteArrayOutputStream();
+        image.compress(Bitmap.CompressFormat.JPEG, QUALITY, stream);
+        byte[] bytes = stream.toByteArray();
+
+        try {
+            stream.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return bytes;
+    }
+
+    /**
+     * Write an image to a file.
+     * @param relativePath The path inside the folder with application data
+     * @param fileName The name of the file that has to be written
+     * @param bytes The bytes that has to be written
+     */
+    public static void writeImage(String relativePath, String fileName, byte[] bytes) {
+        File file = new File(PATH + File.separator + relativePath);
+        file.mkdirs();
+        try {
+            FileOutputStream output = new FileOutputStream(PATH + File.separator + relativePath
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ensure that resources like this FileOutputStream object are closed after use](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5306088)